saleor/userprofile/i18n.py

```diff
@@ -88,6 +93,29 @@ class AddressForm(forms.ModelForm):
     class Meta:
         model = Address
         exclude = []
+        labels = {
+            'first_name': pgettext_lazy(
+                'First name', 'Given name'),
```

Comment:
This is not a "first name" (it will be in some cultures), it's a "person's name" (which sets apart Polish "imię" from "nazwa"). Same for "family name" below.